libraries/WiFi/examples/WiFiClient/WiFiClient.ino

```diff
@@ -1,29 +1,46 @@
 /*
- *  This sketch sends data via HTTP GET requests to data.sparkfun.com service.
- *
- *  You need to get streamId and privateKey at data.sparkfun.com and paste them
- *  below. Or just customize this script to talk to other HTTP servers.
- *
+    Go to thingspeak.com and create an account if you don't have one already.
+    After logging in, click on the "New Channel" button to create a new channel for your data. This is where your data will be stored and displayed.
+    Fill in the Name, Description, and other fields for your channel as desired, then click the "Save Channel" button.
+    Take note of the "Write API Key" located in the "API keys" tab, this is the key you will use to send data to your channel.
+    Replace the channelID from tab "Channel Settings" and privateKey with "Read API Keys" from "API Keys" tab.
+    Replace the host variable with the thingspeak server hostname "api.thingspeak.com"
+    Upload the sketch to your ESP32 board and make sure that the board is connected to the internet. The ESP32 should now send data to your Thingspeak channel at the intervals specified by the loop function.
+    Go to the channel view page on thingspeak and check the "Field1" for the new incoming data.
+    You can use the data visualization and analysis tools provided by Thingspeak to display and process your data in various ways.
+    Please note, that Thingspeak accepts only integer values.
+
+    You can later check the values at https://thingspeak.com/channels/2005329
+    Please note that this public channel can be accessed by anyone and it is possible that more people will write their values.
  */
 
 #include <WiFi.h>
 
 const char* ssid     = "your-ssid";
 const char* password = "your-password";
 
-const char* host = "data.sparkfun.com";
-const char* streamId   = "....................";
-const char* privateKey = "....................";
+const char* host = "api.thingspeak.com";
+const int httpPort = 80;
+const String channelID   = "2005329";
```

Comment:
Add some comments that these are the fields to be changed in case of personal account usage.